### ISSUE TITLE
Custom nets do way too much damage

### ISSUE THREAD

**Issue by Ealb** [elmokki/nationgen#500]
The #dmg for nets is a programming magic number rather than an actual damage value, so if nets get picked to be made magical, they end up doing an obscene amount of damage instead of capturing targets.

**Comment by Ealb** [comment]
I did an update to fix this, committed it, and uploaded it. However - and this is maddening - it appears I've screwed up my repository by starting the commit title with "#". The updated files are in the repository, but the commit is not listed and making a pull request to push the changes to master does nothing because "there are no changes". Worse, it appears all further commits are being ignored. I am assuming that octothorpes being used as comment/identifier characters by GitHub has screwed up the list of commits, and GitHub thinks there aren't any. Grr.
